### DISCUSSION TITLE
I need answer

### DISCUSSION THREAD

**Discussion by Tuanzer** [termux/termux-app#3730] [Q&A]
How i fix :
/data/data/com.termux/files/usr/lib/php

**Comment by TomJo2000** [comment]
It might help if you went into actual detail about the problem you are experiencing.
GitHub issues unfortunately have not yet implemented telepathy or another form of mind reading.

From the file path you have provided it looks like you are facing some sort of issue related to PHP, however there is no possible way to solve your issue if you do not explain what you are having trouble with.

Additionally this is the repository for the Termux app itself, questions related to packages should be asked over in the [termux/termux-packages](https://github.com/termux/termux-packages) repository, where the package builds for all packages available on Termux are maintained.

**Reply by agnostic-apollo** [reply]
>GitHub issues unfortunately have not yet implemented telepathy or another form of mind reading

Powered by JavaScript

**Reply by Tuanzer** [reply]
I use it on the mobileapp:(
